tests/scripts/combined-coverage.sh

```diff
@@ -10,3 +10,5 @@ cp coverage/visual/coverage-storybook.json coverage/merge/coverage-storybook.jso
 
 nyc report --reporter=lcov --reporter=text -t coverage/merge --report-dir coverage/merge
 cp coverage/merge/lcov.info coverage/lcov.info
+
+exit 0
```

Comment:
Is this last exit 0 necessary?

Reply:
The last exit 0 is specific to getting the `slap publish` command to work. Now that there is an exit it will move on to the next check, currently it hangs

Reply:
why does it hang? without the last exit 0, I would still expect this script to end

Reply:
I thought it was last week, but ran again and I didn't need it